src/App.css

```diff
@@ -13,17 +13,6 @@
   }
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
```

Comment:
Inspect the code and remove the rest of the css that you don't need here too. Don't worry, Bootstrap's CSS will probably cover most of what you'd want there so you might be able to just delete it completely